auth/build.gradle

```diff
@@ -15,9 +15,15 @@ android {
 
     buildTypes {
         release {
+            manifestPlaceholders = [enableFbLogging: true]
+
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
+
+        debug {
+            manifestPlaceholders = [enableFbLogging: false]
```

Comment:
This is fine, just add a comment here saying this is only for testing.  Since in the real world library modules are always compiled in their 'release' variant.  So our end users will see `enableFbLogging=true` always